server/game/cards/locations/06/thetumblestone.js

```diff
@@ -0,0 +1,28 @@
+const DrawCard = require('../../../drawcard.js');
+
+class TheTumblestone extends DrawCard {
+    setupCardAbilities() {  
+        this.reaction({
+            when: {
+                // Currently has false positive when power is moved to a character, should only trigger on 'gains'
+                onCardPowerChanged: (event, card, power) => {
+                    if(!card.hasTrait('House Tully') || card.getType() !== 'character' || power === 0 || this.tokens['gold'] === 0) {
+                        return false;
+                    }
+                    
+                    this.standCard = card;
+                    return true;
+                }
+            },
+            handler: () => {
+                this.removeToken('gold', 1);
```

Comment:
This should be a cost, though we don't have a 'removeToken' cost currently. if you're not comfortable writing a new type of cost, that's fine, add a TODO comment here so we don't forget about it.